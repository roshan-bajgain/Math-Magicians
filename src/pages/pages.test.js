// Test for pages/Home.js
import React from 'react';
import { render } from '@testing-library/react';
import Home from './Home';
import Quote from './Quote';

it('renders correctly', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});

it('renders correctly', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment()).toMatchSnapshot();
});
