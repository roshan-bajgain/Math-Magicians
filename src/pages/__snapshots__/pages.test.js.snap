// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h2>
      Welcome to my page
    </h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde rem sit quisquam commodi accusantium itaque quam praesentium quas, libero cumque quasi inventore molestias tenetur laboriosam possimus blanditiis amet corporis eveniet odit? Vitae veniam reiciendis consectetur ipsa error perferendis fugit, quas numquam corrupti ipsum dolorem consequatur saepe debitis voluptates sed ratione iure magni cum harum. Asperiores, voluptatibus excepturi impedit sint pariatur tenetur, repellendus eligendi quas ullam ducimus libero totam mollitia? In illo fuga laboriosam asperiores temporibus consequatur, sit velit voluptatem. Consectetur maiores, non neque qui sequi autem quibusdam quo tenetur! Quos blanditiis aut explicabo harum minima deleniti nam quod totam corrupti!
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde rem sit quisquam commodi accusantium itaque quam praesentium quas, libero cumque quasi inventore molestias tenetur laboriosam possimus blanditiis amet corporis eveniet odit? Vitae veniam reiciendis consectetur ipsa error perferendis fugit, quas numquam corrupti ipsum dolorem consequatur saepe debitis voluptates sed ratione iure magni cum harum. Asperiores, voluptatibus excepturi impedit sint pariatur tenetur, repellendus eligendi quas ullam ducimus libero totam mollitia? In illo fuga laboriosam asperiores temporibus consequatur, sit velit voluptatem. Consectetur maiores, non neque qui sequi autem quibusdam quo tenetur! Quos blanditiis aut explicabo harum minima deleniti nam quod totam corrupti!
    </p>
  </div>
</DocumentFragment>
`;

exports[`renders correctly 2`] = `
<DocumentFragment>
  <div
    class="quotes-paragraph"
  >
    <p>
      Mathematics is not about number, equations,computation,
    </p>
    <p>
      or algorith: It is about understanding. _ William paul thurston
    </p>
  </div>
</DocumentFragment>
`;
